backend/domain/DTO/request/CreateLobbyDTO.js

```diff
@@ -0,0 +1,5 @@
+function CreateLobbyDTO(nickname){
```

Comment:
what exactly does this function do?

is this the dto that is transferred to the lobby/server from the client containing the client's nickname?

Reply:
But isn’t this just a name and and code ..which are both strings? So why not just use them as they are ? Like why create this DTO?

Reply:
Sorry to be annoying I get it now.. but it seems like you are using it like a class? Do maybe update to es6 classes for consistency ?